shell/components/SortableTable/index.vue

```diff
@@ -518,6 +525,23 @@ export default {
       immediate: true
     },
   },
+  setup(_props, { emit }) {
+    const table = useTemplateRef('table');
+
+    const handleEnterKey = (event) => {
+      if (event.key === 'Enter' && !event.explicitOriginalTarget.classList.contains('checkbox-custom')) {
```

Comment:
This yields an error for me:
<img width="2160" alt="Screenshot 2025-02-12 at 09 56 11" src="https://github.com/user-attachments/assets/799d0b83-4876-4a28-8d60-53db768c0919" />

I didn't know about `explicitOriginalTarget` property... Apparently in non-standard and that's probably why this is not working on Chrome.

May I suggest we do `!event.target?.classList?.contains` instead?